elasticapm/traces.py

```diff
@@ -1029,9 +1028,8 @@ def label(**labels):
     """
     transaction = execution_context.get_transaction()
     if not transaction:
-        error_logger.warning("Ignored labels %s. No transaction currently active.", ", ".join(labels.keys()))
-    else:
-        transaction.label(**labels)
+        return
+    transaction.label(**labels)
```

Comment:
I think removing the error log completely is probably not the right move. Instead, can we check if the agent is enabled, and if so, then we can log the error? I think warnings are useful when we're throwing away data unexpectedly -- but I agree that it's not unexpected if the agent is `enabled=False`.

You can get the config from `transaction._tracer.config.enabled`.

Reply:
Fair point. I'm gonna change it accordingly.

Reply:
In case of a disabled agent there is no transaction and thus your suggestion doesn't work. Do you have any other recommendation to retrieve the config? Maybe via `get_client()`?